mysql-test/mroonga/storage/collation/utf8mb4_uca1400_as_ci/r/japanese_accent.result

```diff
@@ -0,0 +1,24 @@
+DROP TABLE IF EXISTS diaries;
+SET NAMES utf8mb4;
+CREATE TABLE diaries (
+content varchar(256) COLLATE utf8mb4_uca1400_as_ci,
+FULLTEXT INDEX (content)
+) DEFAULT CHARSET=utf8mb4;
+INSERT INTO diaries VALUES ('はひふへほ');
+INSERT INTO diaries VALUES ('ほへふひは');
```

Comment:
Can we remove reverse data?

Reply:
fix: [dcacda6](https://github.com/mroonga/mroonga/pull/910/commits/dcacda67b92ae1e398b1857ff11bd1b0444f7a63) Unlike the test case for `utf8mb4_uca1400_ai_ci`, these data are meaningless in this test. So I deleted them.